flink-metrics/flink-metrics-graphite/pom.xml

```diff
@@ -62,20 +62,26 @@ under the License.
 	<build>
 		<plugins>
 			<plugin>
-				<artifactId>maven-assembly-plugin</artifactId>
-				<version>2.4</version>
-				<configuration>
-					<descriptorRefs>
-						<descriptorRef>jar-with-dependencies</descriptorRef>
-					</descriptorRefs>
-				</configuration>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<version>2.4.1</version>
 				<executions>
+					<!-- Run shade goal on package phase -->
 					<execution>
-						<id>make-assembly</id>
+						<id>shade-flink</id>
 						<phase>package</phase>
 						<goals>
-							<goal>single</goal>
+							<goal>shade</goal>
 						</goals>
+						<configuration combine.self="override">
```

Comment:
Why do you use `override` instead of `append` here? Override will disable the automatic reduced pom file generation.